Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -45,11 +45,33 @@ public class GraphLinter: GraphLinting {
         issues.append(contentsOf: lintMismatchingConfigurations(graphTraverser: graphTraverser))
         issues.append(contentsOf: lintWatchBundleIndentifiers(graphTraverser: graphTraverser))
         issues.append(contentsOf: lintCodeCoverageMode(graphTraverser: graphTraverser))
+        issues.append(contentsOf: lintWorkspaceUnuesedTargetsInWorkspace(graphTraverser: graphTraverser))
```

Comment:
Note: schemes can also be defined within `Project`s too, the check would need to be expanded to to accommodate those as well for completeness.

Reply:
I used function schemes() of GraphTraverser, which as far I understand find all schemes in graph